tests/benchmarks/test_subscriptions.py

```diff
@@ -24,3 +25,23 @@ async def _run():
             assert value.data["something"] == "Hello World!"
 
     benchmark(lambda: asyncio.run(_run()))
```

Comment:
This did regress a lot. Is that expected? Any way to improve that?

Reply:
Yeah probably expected, because it tests the lifespan part (which got significantly more complicated).
It is expected to be much closer in the new benchmark I added.

Reply:
can we move that benchmark to another PR? so we can see the new difference :)

Reply:
[#3593](https://github.com/strawberry-graphql/strawberry/pull/3593)

Reply:
![image](https://github.com/user-attachments/assets/2f4d58e0-47d9-4a17-b5bb-773dfba0b43f)
VS
![image](https://github.com/user-attachments/assets/dbfd7e86-0593-4bf4-bc9b-f230021cbf9e)